utopia-remix/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "utopia-remix",
+  "private": true,
+  "sideEffects": false,
+  "type": "module",
+  "scripts": {
+    "build": "remix build",
+    "dev": "remix dev --manual",
+    "lint": "eslint --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint .",
+    "start": "remix-serve ./build/index.js",
+    "typecheck": "tsc",
+    "format": "prettier --write ."
+  },
+  "prettier": {},
+  "dependencies": {
+    "@remix-run/css-bundle": "^2.5.1",
```

Comment:
Please can you remove all of the `^` prefixes from these dependencies so that we don't end up with build discrepancies between team members

Reply:
https://github.com/concrete-utopia/utopia/pull/4807/commits/799a48208aea95a6c1412f5ecb72781dae3e8321